src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -209,7 +212,7 @@ def evaluate(
     data: Optional[str] = None,
     evaluators: Optional[Dict[str, Callable]] = None,
     evaluator_config: Optional[Dict[str, Dict[str, str]]] = {},
-    tracking_uri: Optional[str] = None,
+    azure_ai_project: Optional[Dict] = None,

```

Comment:
can we modify the e2e test to cover this?

Reply:
Done